cmd/fauxrpc/cmd_curl.go

```diff
@@ -0,0 +1,248 @@
+package main
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	"log/slog"
+	"net"
+	"net/http"
+	"strings"
+
+	"connectrpc.com/connect"
+	"github.com/brianvoe/gofakeit/v7"
+	"github.com/quic-go/quic-go/http3"
+	"github.com/sudorandom/fauxrpc"
+	"github.com/sudorandom/fauxrpc/private/server"
+	"github.com/sudorandom/fauxrpc/private/stubs"
+	"github.com/sudorandom/fauxrpc/protocel"
+
+	"golang.org/x/net/http2"
+	"google.golang.org/protobuf/encoding/protojson"
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/reflect/protoreflect"
+	"google.golang.org/protobuf/types/dynamicpb"
+)
+
+type CurlCmd struct {
+	Schema       []string `help:"RPC schema modules."`
+	Addr         string   `short:"a" help:"Address to make a request to" default:"http://127.0.0.1:6660"`
+	NoReflection bool     `help:"Disables server reflection."`
```

Comment:
Is this option even needed? Either users provide schema, and the schema is used or the user doesn't provide the schema and server reflection is used. I don't think this option is really needed.